src/test/java/org/folio/consortia/service/SharingPolicyServiceTest.java

```diff
@@ -62,10 +63,11 @@ void shouldStartSharingPolicy() {
     var expectedPubRequestPut = createPublicationRequest(CONSORTIUM.getPolicyValue(), payload, HttpMethod.PUT)
       .tenants(Set.of(TENANT_ID_1))
       .url(request.getUrl() + "/" + request.getPolicyId());
```

Comment:
`String.format("%s/%s", request.getUrl(), request.getPolicyId())` can be nicer

Reply:
Good idea, but From my perspective, current state is more simpler to read in tests and help to explain situtation more easier, so keeping it current format